Readme.md

```diff
@@ -3,26 +3,21 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T827941)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* [Index.razor](./CS/DxBlazorComponentsDefaultSettings/Pages/Index.razor)
-* [MyTextBox.cs](./CS/DxBlazorComponentsDefaultSettings/Components/MyTextBox.cs)
-* [MyDataGrid.cs](./CS/DxBlazorComponentsDefaultSettings/Components/MyDataGrid.cs)
-<!-- default file list end -->
-
-### DevExpress Blazor Components - A way to set predefined settings for the specific component
+# DevExpress Blazor Components - Set predefined settings for the specific component
 
-In the case of a large Web application, you often use controls with the same settings on numerous pages.  It's quite annoying to specify the same settings many times.
-In this example we show how to specify some default settings for all DevExpress Blazor components in your application.
+In case of a large web application, controls usually have the same settings on different pages. The recommended Blazor approach is to specify the component settings in the markup. It is possible to override the **SetParametersAsync** method to specify default settings for Blazor components in your application. However, we do not recommend that you use this approach as it is not natural for Blazor applications. We also do not test this approach to specify predefined settings with all our controls, so you can use it with your own risk.
 
-Create an inheritor from our Blazor control and override the *SetParametersAsync* method. In this method, set the necessary default values for the **public** properties before calling the base method. Since the base method is called after your custom settings, you can initialize properties with custom default values in markup.
+![Grid with predefined settings and default](images/result.png)
 
-In this example, the [MyDataGrid](./CS/DxBlazorComponentsDefaultSettings/Components/MyDataGrid.cs) component contains the inherited DxDataGrid component whose *PageSize*, *ShowFilterRow*, and *ShowGroupPanel* properties contain new default values. Compare these components with the default DxDataGrid component on the [Index](./CS/DxBlazorComponentsDefaultSettings/Pages/Index.razor) page: they have the same settings in markup.
+Create a component that inherits from our Blazor control and overrides the **SetParametersAsync** method. In this method, set the necessary default values for the **public** properties before you call the base method. You can initialize properties with custom default values in the markup because the base method is called after your custom settings. We do not recommend that you override and change values of any protected properties.

```

Comment:
- `that inherits` Inherits what?
- `the necessary default values` Define **necessary** 
- in the markup -> in markup

Reply:
```suggestion
Create an inheritor from our Blazor control that overrides the **SetParametersAsync** method. In this method, set the desired default values for the **public** properties before you call the base method. You can initialize properties with custom default values in markup because the base method is called after your custom settings. We do not recommend that you override and change values of any protected properties.

```

Reply:
Create an inheritor from our Blazor control that overrides the **SetParametersAsync** method. In this method, set the default values for the **public** properties before you call the base method. You can initialize properties with custom default values in markup because the base method is called after your custom settings. We do not recommend that you override and change the values of protected properties.

Reply:
```suggestion
Create an inheritor from our Blazor control that overrides the **SetParametersAsync** method. In this method, set the default values for the public properties before you call the base method. You can initialize properties with custom default values in markup because the base method is called after your custom settings. We do not recommend that you override and change the values of protected properties.

```